deploy/vagrant/provision/roles/ufs_swift/files/config_alluxio.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cat >> ~/.bashrc << EOF
+export ALLUXIO_UNDERFS_ADDRESS="swift://${SWIFT_CONTAINER}"
+
+export ALLUXIO_JAVA_OPTS+="
+  -Dfs.swift.user=${SWIFT_USER}
+  -Dfs.swift.tenant=${SWIFT_TENANT}
+  -Dfs.swift.password=${SWIFT_PASSWORD}
+  -Dfs.swift.auth.url=${SWIFT_AUTH_URL}
+  -Dfs.swift.use.public.url=${SWIFT_USE_PUBLIC_URL}
+  -Dfs.swift.auth.method=${SWIFT_AUTH_METHOD}
+"
```

Comment:
Is it possible to use the `alluxio-site.properties` file to set this options? Though I guess the whole deploy module does this, so it can be refactored in another PR.